pkg/ebpf/events_pipeline.go

```diff
@@ -271,15 +368,19 @@ func (t *Tracee) processEvents(ctx context.Context, in <-chan *trace.Event) (<-c
 				continue
 			}
 
-			if (t.config.Filter.ContFilter.Value() || t.config.Filter.NewContFilter.Enabled()) && event.ContainerID == "" {
+			if t.config.FilterScopes.ContainerFilterEnabled() && event.ContainerID == "" {
 				// Don't trace false container positives -
 				// a container filter is set by the user, but this event wasn't originated in a container.
 				// Although kernel filters shouldn't submit such events, we do this check to be on the safe side.
 				// For example, it might be that a new cgroup was created, and not by a container runtime,
 				// while we still didn't processed the cgroup_mkdir event and removed the cgroupid from the bpf container map.
-				id := events.ID(event.EventID)
-				// don't skip cgroup_mkdir and cgroup_rmdir so we can derive container_create and container_remove events
-				if id != events.CgroupMkdir && id != events.CgroupRmdir {
+				eventId := events.ID(event.EventID)
+				logger.Debug("false container positive - possible", "event.Timestamp", event.Timestamp, "eventId", eventId)
+
+				// don't skip cgroup_mkdir, cgroup_rmdir and security_sb_mount so we can derive container_create and container_remove events
+				if eventId == events.CgroupMkdir || eventId == events.CgroupRmdir || eventId == events.SecuritySbMount {
```

Comment:
It used to be a commit (now squashed) related to an possible issue:

```
pipeline: processEvents check for SecuritySbMount

From ubuntu jammy the SecuritySbMount check is required to treat
container false positives.
```

kerneltests were failing in jammy above and I found out that in jammy that was the cause. It's a workaround to let the kerneltests pass and, for sure, need more investigation.

Reply:
I believe this could be related to a recent fix made by @NDStrahilevitz (https://github.com/aquasecurity/tracee/commit/de1bee930fe161c16b5f034bc1999eff96dbb2e3)